src/lib/holocene/date-picker.svelte

```diff
@@ -93,18 +97,21 @@
     >
       <div class="mx-3 my-2 flex items-center justify-around">
         <div class="flex items-center justify-center">
-          <button type="button" on:click={prev}
-            ><Icon name="chevron-left" /></button
+          <button type="button" on:click={prev} title={previousMonth}>
+            <span class="sr-only">{previousMonth}</span>
```

Comment:
Is this preferred over just specifying `aria-label` on the button? I imagine it accomplishing the same thing.